src/utils/Person.ts

```diff
@@ -0,0 +1,11 @@
+import { Entity, BaseEntity, Column} from "typeorm";
+
+@Entity()
+export class Person extends BaseEntity {
```

Comment:
why ? what about you only have one table for all of the users in the system and we differentiate between them with the Role_id ?

Reply:
I used to use this class for testing purposes, so should i remove it?